src/lib/y2network/connection_config/base.rb

```diff
@@ -25,7 +25,8 @@ module ConnectionConfig
     class Base
       # @return [Interface, nil]
       attr_accessor :interface
-      # @return [String] Bootproto (static, dhcp, none)
+      # @return [String] Bootproto (static, dhcp, ,dhcp4, dhcp6, autoip,
+      #   dhcp+autoip, auto6, 6to4, none)
```

Comment:
interesting, we do not support in ui 6to4 and it will fail there. Probably something to fix?

Reply:
And there are more interfaces that are supported by wicked but not by YaST yet, so would be nice to have but step by step.

Reply:
well, issue is that now if user has this configuration it will crash. We probably need to introduce something like bootloader UnsupportedConfiguration exception that allow user to repropose configuration or keep it.

Reply:
Have you tried?, If that is the case then definitely.